### ISSUE TITLE
Initial Calibration: Clear Deck Modal

### ISSUE THREAD

**Issue by umbhau** [Opentrons/opentrons#1236]
As a user, I would like to be informed that I need to clear the deck before starting calibration

## Acceptance Criteria
- [x] When user clicks 'Calibrate' button, and after user takes control of robot (has token), and after we confirm that user has a pipette,
- [x] Inform user that they need to clear tipracks and labware from deck
- [x] User can click 'cancel' and return to robot settings page
- [x] There is a 'continue' button (behavior in a different ticket)

## Implementation Notes
- Include some route work

## Design
- Reuse existing dialogue box (does not match existing screens; restyling out of scope)
